File: Prime Number/Prime.java
import java.util.*;



public class Prime{

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int x,i,f;
        System.out.print("Enter the No:- ");
        x=sc.nextInt();
        sc.close();
        double sq =Math.sqrt(x);
        f=0;
        for(i=2;i<=sq;i++){
            if(x%i==0){
                f=1;
                break;
            }
        }
        if(f==1){
            System.out.println("0.00");
        }
        else{
            System.out.printf("%.2f\n",sq);
        }

        
    }
}

File: Prime Number/prime.cpp
#include<iostream>
#include<math.h>

using namespace std;

int main(){

    int x, i,f;
    f=0;
    cout << "Enter the value of x: ";
    cin >>  x;
    float sq= sqrt(x);
    
    for(i=2;i<=sq;i++){
        if(x%i == 0){
            f=1;
            break;
        }
    }
    if(f>=1){
        cout << "\n0.00";
    }
    else{

        printf("\n%.2f\n", sq);
        // cout << sq <<"\n";
    }


    return 0;
}